src/services/github/user.ts

```diff
@@ -1,17 +1,17 @@
 import { GitHubAPI } from "probot";
 import { Octokit } from "@octokit/rest";
 import { getLogger } from "config/logger";
-import { GitHubAppClient } from "../../github/client/github-app-client";
+import { GitHubInstallationClient } from "../../github/client/github-installation-client";
 
 const logger = getLogger("services.github.user");
 // TODO: Remove this method on featureFlag cleanup
-export const getGithubUser = async (github: GitHubAPI | GitHubAppClient, username: string): Promise<Octokit.UsersGetByUsernameResponse | undefined> => {
+export const getGithubUser = async (github: GitHubAPI | GitHubInstallationClient, username: string): Promise<Octokit.UsersGetByUsernameResponse | undefined> => {
```

Comment:
Do we still need this the github client check?

Reply:
For now, once the github client PR flag goes it can go too.